contrib/kind.sh

```diff
@@ -494,5 +494,7 @@ if ! kubectl wait -n kube-system --for=condition=ready pods --all --timeout=300s
   exit 1
 fi
 
+docker exec -it ovn-worker ip route
```

Comment:
for debugging ci just add this action
https://github.com/marketplace/actions/debugging-with-tmate
and you can ssh into the instance of the CI for 1 hours